app/controllers/concerns/idv_session.rb

```diff
@@ -8,12 +8,24 @@ module IdvSession
     end
   end
 
+  def confirm_idv_session_started
+    redirect_to idv_session_url unless idv_session[:params].present?
+  end
+
   def idv_session
     user_session[:idv] ||= {}
   end
 
   protected
 
+  def idv_finance_complete?
+    (idv_params.keys & Idv::FinanceForm::FINANCE_TYPES).any?
+  end
+
+  def idv_phone_complete?
+    idv_params[:phone].present?
+  end
```

Comment:
These 2 methods are only called in ReviewController. Move them there since they're not shared?